mgmtsystem_manuals/views/mgmtsystem_system.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+  <data>
```

Comment:
For 9.0  we can use `<odoo>` instead of `<openerp>` and the `<data>` element is optional and can be removed.